apps/core-app/src/main/modules/ai/intelligence-local-environment.test.ts

```
@@ -93,7 +99,70 @@ describe('getIntelligenceLocalEnvironment', () => {
+        'name: Custom Review',
+        'description: Local unreviewed skill.',
+        '---',
+        '# Custom Review'
+      ].join('\n')
+    )
+
+    process.env.CODEX_HOME = codexHome
+    process.env.PATH = ''
+
+    const summary = await getIntelligenceLocalEnvironment(root)
+    const sentry = summary.skillProviders.find((item) => item.id === 'sentry')
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)